packages/fx-core/resource/package.nls.json

```diff
@@ -93,7 +93,7 @@
   "core.migrationToArmAndMultiEnv.Message": "Teams Toolkit will upgrade your project configuration files to support the latest features. The upgrade process will not change your custom code and create the backup files in case revert is needed.\nNotice this upgrade is a must-have to continue to use current version Teams Toolkit. If you are not ready to upgrade and want to continue to use the old version, please find Teams Toolkit in Extension and install the version <= 2.10.0",
   "core.migrationV3.confirmOnly.Message": "Please confirm the upgrade",
   "core.migrationV3.Message": "Upgrade your Teams Toolkit project to stay compatible with the latest version. A backup directory will be created along with an Upgrade Summary.",
-  "core.migrationV3.VS.Message": "The Teams Toolkit preview feature - Teams App Configuration Improvements is turned on. This project is required to be upgraded to work with the preview features. Select \"Learn more\" to read about the required changes or select \"Upgrade\" to perform the changes automatically. If you don't want to upgrade, you can turn off the preview feature.",
+  "core.migrationV3.VS.Message": "Upgrade your solution to stay compatible with the latest Teams Toolkit version. A backup directory will be generated in which contains an upgrade report for you to understand the changes.",
```

Comment:
for you to understand the changes -> delete
contains -> it contains

Reply:
Fixed. Thanks.